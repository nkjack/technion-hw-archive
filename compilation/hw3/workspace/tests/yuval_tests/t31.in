void main(){
  int x;
}
int checkFuncCall(){
  int k;
  k=checkFuncCall(5 b,false,6 b,4 b);
}
