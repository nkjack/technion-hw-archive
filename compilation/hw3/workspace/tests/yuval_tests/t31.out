---end scope---
main - 0 preconditions
x INT 0
line 6: prototype mismatch, function checkFuncCall expects arguments ()
